uisdk/src/test/java/tech/dojo/pay/uisdk/domain/mapper/PaymentIntentDomainEntityMapperTest.kt

```diff
@@ -41,6 +41,30 @@ internal class PaymentIntentDomainEntityMapperTest {
             Assert.assertEquals(expected, actual)
         }
 
+    @Test
+    fun `when calling apply with valid PaymentIntentPayload with status as captured then should map to PaymentIntentDomainEntity with isPaymentAlreadyCollected as true `() =
+        runTest {
+            // arrange
+            val raw = createValidPaymentIntentPayload().copy(status = "Captured")
+            val expected = getValidPaymentIntentDomainEntity().copy(isPaymentAlreadyCollected = true)
+            // act
+            val actual = PaymentIntentDomainEntityMapper().apply(raw)
```

Comment:
you could instantiate only once rather than having to instantiate always manually for each test